src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public interface OperationHandler {
+    void handle(FruitTransaction transaction, Map<String, Integer> storage);
```

Comment:
```suggestion
    void handle(FruitTransaction transaction);
```
Don’t  pass Map as a parameter, call it directly inside the methods